packages/components/src/step/step.tsx

```diff
@@ -8,16 +8,17 @@ import type { ColorTokens } from '@tamagui/core'
 export type StepVariants = 'neutral' | 'complete' | 'active'
 
 type Props = {
+  size: 18 | 22
   value: number
   type?: StepVariants
 }
 
 const Step = (props: Props) => {
-  const { value, type = 'neutral' } = props
+  const { size, value, type = 'neutral' } = props
```

Comment:
Even if it rendered correctly without the default size, it makes sense to have it to avoid passing that prop when using it.

Reply:
@jkbktl could you confirm that's what you meant by your comment and that https://github.com/status-im/status-web/pull/552/commits/911564ac3c3e88594d0c5d1538ec46d3c2d5acd9 resolves it please?

Reply:
Confirmed.